geotrek/feedback/templates/feedback/report_detail_attributes.html

```diff
@@ -125,7 +125,7 @@ <h3>{% trans "Attributes" %}</h3>
             <td colspan="3">
             {% if perms.maintenance.add_intervention %}
                 <h3>{% trans "Intervention" %}</h3>
-                    {% with columns="name,status,stake,total_cost,date" %}
+                    {% with columns="name,status,stake,total_cost,begin_date" %}
```

Comment:
Please display `end_date` here as well (same as in `geotrek/maintenance/templates/maintenance/maintenance_detail_fragment.html`)